rest-server/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pai-rest-server",
-  "version": "0.1.0",
+  "version": "0.8.0",
```

Comment:
I will approve the PR first. 
is it possible to get the version automatically from the release branch name?

Reply:
@fanyangCS Currently there is no third part "y" in branch name "pai-0.7.y", so we can only *validate* the version field on release branches but cannot auto *generate* it.

BTW, I used `git describe HEAD` as temporary version names in some early practices, which includes latest tag, commits count since latest tag and current commit hash. Unfortunately, it is not match our git flow, a little, since our release tags is not on the master branch.

Our `git describe --tags master` now looks like `v0.5.0-822-ga17f1548`, but `git describe --tags v0.7.1` is `v0.7.1`(add `--tags` because we are using lightweight tags but not annotated tags for release versions), does that look fine?